app/src/main/java/com/tip/lunchbox/model/server/request/Comment.java

```diff
@@ -0,0 +1,36 @@
+package com.tip.lunchbox.model.server.request;
+
+import com.google.gson.annotations.Expose;
+
+public class Comment {
+    @Expose
+    private String comment;
+    @Expose
+    private String title;
+    @Expose
+    private String zomatoResId;
```

Comment:
```suggestion
    @SerializedName("zomato_res_id")
    @Expose
    private String zomatoResId;
```

Reply:
Won't make a difference it's automatically converted to snake case

Reply:
How? Have you tested it? I have experienced otherwise, that's why.